Pages/Productpage/productpage.css

```diff
@@ -0,0 +1,120 @@
+:root {
+    --bg-color: rgb(255, 255, 255);
+    --body-bgcolor: rgb(230, 230, 230);
+}
+
+body {
+    background-color: var(--body-bgcolor);
+}
+
+.container {
+    display: grid;
+    gap: 20px;
+    grid-template-columns: repeat(5,1fr);
+    grid-template-areas: 
+    "header header header header header"
+    "sidebar main  main   main   main"
+    "sidebar main  main   main   main";
+}
+
+.sidebar-nav {
+    margin-top: 5rem;
+    grid-area: sidebar;
+    background-color: var(--bg-color);
+    padding: 1rem;
+    margin-left: 1rem;
+    height: fit-content;
+}
+
+.flex {
+    display: flex;
+}
+
+.align-center {
+    align-items: center;
+}
+
+.space-between {
+    justify-content: space-between;
+}
```

Comment:
if you plan to use these classes in all files, maybe make a separate CSS file for such blocks!

Reply:
sidebar-nav here is only for the product page that's why I have deliberately written here. BTW thanks for the suggestion will keep that in mind.